api/apps/api/src/modules/scenario-specification/old/domain/feature.ts

```diff
@@ -0,0 +1,71 @@
+import { FeatureTag } from '@marxan/features/domain';
+import { FeatureSubSet } from './feature-sub-set';
+import { ComputedFeature } from './computed-feature';
+
+/**
+ * TODO may be useful for finding features on geo, thus leaving it
+ */
+export class Feature<T extends FeatureTag> {
```

Comment:
it's a `code` draft, right?

Reply:
maybe should be in another PR then

Reply:
fixed in https://github.com/Vizzuality/marxan-cloud/pull/371/commits/4dbad00ade491ccca947032a295834751eab9b2d (removed)